src/responder/common/responder_packet.h

```diff
@@ -26,6 +26,7 @@
 
 #define SSS_PACKET_MAX_RECV_SIZE 1024
 #define SSS_CERT_PACKET_MAX_RECV_SIZE ( 10 * SSS_PACKET_MAX_RECV_SIZE )
+#define SSS_GSSAPI_PACKET_MAX_RECV_SIZE ( 128 * 1024 )
```

Comment:
According to [docs](https://docs.microsoft.com/en-us/troubleshoot/windows-server/group-policy/group-policy-add-maxtokensize-registry-entry#:~:text=The%20maximum%20allowed%20value%20of,value%20larger%20than%2048000%20bytes.) the token could be up to 64 KiB but the default limit is 48 KiB to prevent a base64-encoded token for being too long for an HTTP header...